### ISSUE TITLE
Lower case vs. CamelCase

### ISSUE THREAD

**Issue by rickj** [w3c/publ-a11y#459]
In the "detailed-Conformance" section of https://github.com/w3c/publ-a11y/blob/main/a11y-meta-display-guide/2.0/draft/localizations/en-US/display_guide_vocabulary_w3c_en-US.json the word 'Conformance' is upper cased in the heading "detailed-Conformance".

My developers are asking if this is intentional (as all the others are lower case), or will it be changing?

**Comment by bvandorn** [comment]
Similarly, `conformance-not-EPUB` should maybe be `conformance-not-epub` (though I can see how that one might be treated differently)

**Comment by GeorgeKerscher** [comment]
In the guidelines (was principles) the heading reads "Detailed Conformance Information" According to our editing guidelines, The first letter of a heading should be capatolized and the other words in the heading are lower case. This is a violation of this editing guideline.

The same problem occurs in the ONIX techniques and in the EPUB techniques.

In the comparison spreadsheet, tthey are marked as true, because it is wrong everywhere.
I can do a PR that resolves this problem.

I believe the JSON should be corrected to read "Detailed conformance information".

**Comment by gregoriopellegrino** [comment]
@GeorgeKerscher the problem I think is with the IDs, not the text content

**Comment by GeorgeKerscher** [comment]
Right, I see the problem. I will fix both the id and the text in all places.

**Comment by GeorgeKerscher** [comment]
I just submitted a PR that fixes some of the problems. It is:
https://github.com/w3c/publ-a11y/pull/463 

The cammel case @rickj  reported must be in the JSON file, because I cannot find it anywhere else.

**Comment by mattgarrish** [comment]
> The cammel case @rickj reported must be in the JSON file, because I cannot find it anywhere else.

Yes, this issue is about the naming of the variables in the file. The fixes you made to the text are in keeping with W3C style, but are not the root of this issue.
